python/pyspark/_typing.pyi

```diff
@@ -21,11 +21,14 @@ from typing_extensions import Protocol
 
 F = TypeVar("F", bound=Callable)
 T_co = TypeVar("T_co", covariant=True)
+T = TypeVar("T", covariant=True)
```

Comment:
Nit, this shouldn't be here.

Reply:
qq: Why do we have both `T_co` and `T` for the same type ??

Reply:
updated. thanks!